File: launchdarkly/typescript-node/src/DeleteWebhookExample.ts
import * as fs from 'fs';
import api from "launchdarkly_client"
import models from "launchdarkly_client"

const apiCaller = new api.WebhooksApi();
apiCaller.setApiKey(api.WebhooksApiApiKeys.ApiKey, "YOUR_API_KEY");

apiCaller.deleteWebhook(
  "id_string", // id
).catch(error => {
  console.log("Exception when calling WebhooksApi#deleteWebhook:");
  console.log(error.body);
});

File: launchdarkly/ruby/src/DeleteWebhookExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    LaunchDarklyClient::WebhooksApi.new.delete_webhook(
        nil, # id
    )
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling WebhooksApi#delete_webhook: #{e}"
end
